src/moveit_commander/interpreter.py

```diff
@@ -557,16 +557,26 @@ def get_help(self):
         res.append("  go <dir> <dx>|      plan and execute a motion in direction up|down|left|right|forward|backward for distance <dx>")
         res.append("  go rand             plan and execute a motion to a random state")
         res.append("  plan <name>         plan a motion to the state <name>")
+        res.append("  plan rand           plan a motion to a random state")
+        res.append("  planner <name>      use planner <name> to plan next motion")
         res.append("  execute             execute a previously computed motion plan")
+        res.append("  stop                stop the active group")
+        res.append("  pick <name>         pick up object <name>")
+        res.append("  place <name>        place object <name>")
+        res.append("  stop                stop any execution, if active")
```

Comment:
Sorry, my comment https://github.com/ros-planning/moveit_commander/pull/42/commits/a3df575bee1d1d22c204203d68ae56fc818ddffb#r69403042 is invalid (I realized that I was looking at a single commit page. You already fixed it in a subsequent commit).